apps/blueman-sendto.in

```diff
@@ -19,120 +18,12 @@ if 'BLUEMAN_SOURCE' in os.environ:
 _ = gettext.gettext
 gettext.textdomain("@GETTEXT_PACKAGE@")
 
-from blueman.Functions import (
-    setup_icon_path,
-    check_bluetooth_status,
-    set_proc_title,
-    create_parser,
-    create_logger,
-    bmexit
-)
-from blueman.main.Sendto import Sender
-from blueman.bluez.Manager import Manager
-from blueman.bluez.errors import DBusNoSuchAdapterError
-from blueman.gui.DeviceSelectorDialog import DeviceSelectorDialog
+from blueman.Functions import set_proc_title, create_parser, create_logger
 
 # Workaround introspection bug, gnome bug 622084
 signal.signal(signal.SIGINT, signal.SIG_DFL)
 
 
-class SendTo:
```

Comment:
This lacks an important now

Reply:
Initially I was very confused what you meant here 😆 . Fixed!